### DISCUSSION TITLE
clip studio paint

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#21] [General]
# clip studio paint

Check if clip studio paint is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/f25d8baf-ad36-4125-a579-192704f5946c

**Comment by OpenSource03** [comment]
> The under review option was not in the dropdown menu, sorry It is a drawing program and I'm curious if it's functional on it

Thanks, it's a known bug, will test soon and let you know :)
